editor/src/components/editor/shortcuts.spec.browser2.tsx

```diff
@@ -60,26 +65,56 @@ describe('shortcuts', () => {
     })
   })
 
-  it('x to remove positioning props from the selected element', async () => {
-    const editor = await renderTestEditorWithCode(project, 'await-first-dom-report')
+  describe('x', () => {
+    it('remove positioning props from the selected element participates in the layout', async () => {
+      const editor = await renderTestEditorWithCode(project, 'await-first-dom-report')
 
-    const canvasControlsLayer = editor.renderedDOM.getByTestId(CanvasControlsContainerID)
-    const div = editor.renderedDOM.getByTestId(TestIdOne)
-    const divBounds = div.getBoundingClientRect()
-    const divCorner = {
-      x: divBounds.x + 50,
-      y: divBounds.y + 40,
-    }
+      const canvasControlsLayer = editor.renderedDOM.getByTestId(CanvasControlsContainerID)
+      const div = editor.renderedDOM.getByTestId(TestIdOne)
+      const divBounds = div.getBoundingClientRect()
+      const divCorner = {
+        x: divBounds.x + 50,
+        y: divBounds.y + 40,
+      }
 
-    mouseClickAtPoint(canvasControlsLayer, divCorner)
+      mouseDoubleClickAtPoint(canvasControlsLayer, divCorner)
 
-    pressKey('x')
+      await expectSingleUndoStep(editor, async () => pressKey('x'))
+      await editor.getDispatchFollowUpActionsFinished()
+
+      expect(div.style.position).toEqual('')
+      expect(div.style.top).toEqual('')
+      expect(div.style.bottom).toEqual('')
+      expect(div.style.left).toEqual('')
+      expect(div.style.right).toEqual('')
+    })
+
+    it('add positioning props from the selected element participates in the layout', async () => {
```

Comment:
something doesn't read right about this test title :) could you make it simpler english?

Reply:
😬 fixed in [1c32738](https://github.com/concrete-utopia/utopia/pull/3203/commits/1c3273820ad845c07692c0cc3594ba13d99c6590)